src/features/DeFiZap/components/ZapCard.tsx

```diff
@@ -300,12 +303,16 @@ const ZapCard = ({ config }: Props) => {
       <ZapCardContentBottom>
         {!humanReadableZapBalance ? (
           <RouterLink to={`${ROUTE_PATHS.DEFIZAP.path}/zap?key=${config.key}`}>
-            <ZapCardButton inverted={true}>{translateRaw('ZAP_CARD_CTA')}</ZapCardButton>
+            <ZapCardButton colorScheme={TButtonColorScheme.inverted}>
```

Comment:
Can we do it without having to import `TButtonColorScheme` every time? (Goes for every file changed)